### PR TITLE
PS-5179 Document other assorted changes because of upstream 8.0.13 merge

### PR DESCRIPTION
**Pull request by patrickbirch**
added text to diagnostics/innodb_show_status
added text to index
added management/backup_locks
added management/changed_page_tracking
added management/enforce_engine
added text to source/scalability/innodb_io

### CODE REVIEW

File: doc/source/diagnostics/innodb_show_status.rst [hunk 1]
```diff
@@ -224,10 +224,15 @@ between the current LSN and the LSN of the last completed checkpoint.
...
 
-This variable shows the maximum allowed checkppoint age above which the redo
+This variable shows the maximum allowed checkpoint age above which the redo
 log is close to full and a checkpoint must happen before any further redo log
 writes.
 
+.. note:: 
+
+        This variable was removed in |Percona Server| 8.0.13-4 due to a change
+        in MySQL. The variable is identical to log capacity.
```
Comment (dutow) [inline doc/source/diagnostics/innodb_show_status.rst:234]: "to log capacity."?

File: doc/source/management/enforce_engine.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.. _enforce_engine:
+
+========================
+Enforcing Storage Engine
+========================
+
+|Percona Server| has implemented variable which can be used for enforcing the
```
Comment (dutow) [inline doc/source/management/enforce_engine.rst:7]: "has implemented the `enforce_storage_engine`" variable? The name isn't mentioned at all until the bottom of this page.

File: doc/source/diagnostics/innodb_show_status.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/diagnostics/innodb_show_status.rst:234]: From the description of the issue:
No longer prints maximum checkpoint age in SHOW ENGINE INNODB STATUS extensions due to it being made identical to log capacity after upstream 28072385

Thanks.

File: doc/source/management/enforce_engine.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/management/enforce_engine.rst:7]: Fixed. Added the variable name to the first sentence.

File: doc/source/diagnostics/innodb_show_status.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/diagnostics/innodb_show_status.rst:234]: Checked with Yura - he had nothing to add and said that Zsolt's comments covered everything.
Yura is ok with the current sentence - "The variable is identical to log capacity."

**Comment by patrickbirch** [comment]
waiting for comments or approval.

**Comment by george-lorch** [comment]
@dutow @percona-ysorokin ping, approved?

**Review by dutow — APPROVED** [review]

**Pull request merged**